src/screens/Tank/TankWithdrawal/TankWithdrawal.js

```diff
@@ -241,7 +254,7 @@ class TankWithdrawal extends React.Component<Props, State> {
     );
     const formFields = getAmountFormFields({
       icon: iconUrl,
-      currency: token,
+      currency: symbol,
```

Comment:
mixing currency and symbols? something does not feel right here.

Reply:
No idea mate, nothing changed here except naming. Seems like `getAmountFormFields` is used only on PPN flows and all of the calls have `currency`. Will check deeper, but I'm leaning to not touch this at all (except the rename part).

Reply:
I don't think `currency` is used at all, it's passed as `config`, but never destructured from it, only `icon` from these calls.